init.d/codedeploy-agent

```diff
@@ -17,11 +17,14 @@
 #                    the deployment artifacts on to this instance.
 ### END INIT INFO
 
+# Source function library.
+. /etc/rc.d/init.d/functions
```

Comment:
Since this is already merged, I will add a new PR with logic to check for RHEL 7.  Will that suffice?